waves-ext/src/main/resources/application.conf

```diff
@@ -1,11 +1,10 @@
-# To enable this extension on NODE, add this line:
-# waves.extensions += "com.wavesplatform.dex.grpc.integration.DEXExtension"
+waves.extensions += "com.wavesplatform.dex.grpc.integration.DEXExtension"
 
 waves.dex {
   # gRPC integration settings for Waves Node
   grpc.integration {
     # Extension's host
-    host = localhost
+    host = "0.0.0.0"
```

Comment:
It's better to do this only for a docker image. Probably this way: https://github.com/lightbend/config#optional-system-or-env-variable-overrides
> By setting the JVM property -Dconfig.override_with_env_vars=true it is possible to override any configuration value using environment variables even if an explicit substitution is not specified.

Reply:
Done